src/frameworks/angular/utils.ts

```diff
@@ -79,10 +79,21 @@ const enum ExpectedBuilder {
   LEGACY_DEVKIT_SSR_DEV_SERVER = "@angular-devkit/build-angular:ssr-dev-server",
 }
 
+const enum ThirdPartyBuilder {
```

Comment:
Qq, could we just simplify all this and only look after the last colon and expect it to conform to the builder-type? e.g, ":application", ":dev-server", ":server", ":browser", etc.